apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/HealthResource.java

```diff
@@ -22,13 +22,20 @@
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 
+/** Legacy Health endpoint used by some routers to check if the service is up and running. */
 @Path("/")
-@Produces(MediaType.APPLICATION_JSON)
+@Produces(MediaType.TEXT_PLAIN)
```

Comment:
Was copied along from v1, but since we do not produce JSON but plain text seems more correct.